lib/elements/validation.py

```diff
@@ -39,3 +39,13 @@ class ValidationWarning(ValidationMessage):
     don't, Ink/Stitch will do its best to process the object.
     """
     pass
+
+
+class ValidationTypeWarning(ValidationMessage):
```

Comment:
How about just `ObjectTypeWarning`?  At first I was confused wondering what a "Validation Type" is.

Reply:
Hehe, oh yes. I am bad at naming things ;)

Reply:
> There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors. -- Leon Bambrick

;)